html-forms/index.html

```diff
@@ -21,11 +21,13 @@ <h1>Coding Form!</h1>
         </select>
       </div>
       <div>
-        <label for="coding language"> Favorite Language (Choose one) </label>
-        <input type="radio" name="coding language" value="HTML" /> HTML
-        <input type="radio" name="coding language" value="CSS" /> CSS
-        <input type="radio" name="coding language" value="JavaScript" />
-        Javascipt
+        <label for="coding language">
+          Favorite Language (Choose one)
+          <input type="radio" name="coding language" value="HTML" /> HTML
+          <input type="radio" name="coding language" value="CSS" /> CSS
+          <input type="radio" name="coding language" value="JavaScript" />
+          Javascipt
```

Comment:
Each of these pieces of text needs to be in its own individual label so that it can pair with its form control element.